src/components/one_click_breakdown/OneClickBreakdown.jsx

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable react/forbid-prop-types */
```

Comment:
This has been removed

Reply:
The breakdown component in the quick checkout figma is quite a bit different than the vanilla Breakdown component. That's why I created this new OneClickBreakdown file. I could change the look of the Breakdown component to match the one in the figma file. That way we would not need this separate OneClickBreakdown component.